HT5.py

```diff
@@ -0,0 +1,111 @@
+import datetime
+import random
+
+
+class Publication:
+    def __init__(self, name):
+        self.name = name
+
+    def get_publication_text(self):
+        txt = input('Type text: ')
+        return txt
+
+    def write_to_list(self):
+        pub_list.insert(0, pub.name)
+        pub_list.insert(1, self.get_publication_text())
+        return pub_list
+
+    def write_to_file(self, l1):
+        file = open('myfile.txt', 'a+')
+        file.write('---------------------------------' + '\n')
+        file.write(l1[0] + '\n' + l1[1] + '\n' + l1[2] + '\n')

```

Comment:
try to use f-string for concatenation
`f"{l1[0]}\nl1[1]..."`

Reply:
Ok